internal/postprocessor/go.mod

```diff
@@ -1,6 +1,8 @@
 module cloud.google.com/go/internal/postprocessor
 
-go 1.20
+go 1.21
+
+toolchain go1.22.0
```

Comment:
We should figure out what dep requires 1.22 and remove it or downgrade for now

Reply:
ugh

Reply:
I think if you bump this to 1.21 you can remove the toolchain

Reply:
I re-ran with 1.20